app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -22,10 +29,11 @@ const ModalWizard = props => {
   const onFirstStep = activeStepIndex === 0;
   const onFinalStep = activeStepIndex === numSteps - 1;
 
-  const formHasErrors =
-    find(formContainer, form => form.syncErrors) !== undefined;
-
-  const disableNextStep = formHasErrors;
+  const currentReduxForm = reduxFormMap[title][activeStepIndex];
+  const disableNextStep =
+    formContainer &&
+    Object.prototype.hasOwnProperty.call(formContainer, currentReduxForm) &&
+    !!formContainer[currentReduxForm].syncErrors;
```

Comment:
Yeah, after reading through this, I think it would be better to not have any redux-form specific code in `ModalWizard` either. Let's pass in `disableNextStep` as a prop from above, and lift all this redux-form specific stuff out.